docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -360,13 +381,15 @@ public class ExpoNativeConfigurationModule: Module {
 With your native modules reading the fields added to the native files, you can now run the example app and access your custom API key using the `ExamplePlugin.getApiKey()` function.
 
 <Terminal
-  cmdCopy="cd example && npx expo run:ios"
   cmd={[
     '$ cd example',
-    '# execute our plugin and update native files',
+    '',
+    '# Execute our plugin and update native files',
```

Comment:
```suggestion
    '# Execute your plugin and update native files',
```

Something I forgot to fix but it caught my eye just now. Going to commit before merging this PR.